src/components/react_canvas/reactive_chart.tsx

```diff
@@ -332,8 +332,17 @@ class Chart extends React.Component<ReactiveChartProps, ReactiveChartState> {
       chartRotation,
       chartTransform,
       setCursorPosition,
+      isChartEmpty,
     } = this.props.chartStore!;
 
+    if (isChartEmpty) {
+      return (
+        <div className="euiEmptyPrompt euiSeriesChartContainer__emptyPrompt">
+          {<span className="euiTitle euiTitle--large" style={{paddingBottom: '1.2rem'}}>Chart not available</span>}
```

Comment:
I don't think you need the `{}` around the template code

```suggestion
          <span className="euiTitle euiTitle--large" style={{paddingBottom: '1.2rem'}}>Chart not available</span>
```

Reply:
Thanks! I've revised this a bit in commit ae929db to avoid bringing in EUI styling in this part. Let me know what you think